ibis/mapd/client.py

```diff
@@ -167,6 +185,44 @@ def __exit__(self, exc_type, exc_value, traceback):
         pass
 
 
+class MapDGeoCursor(MapDDefaultCursor):
+    """Cursor to allow the MapD client to reuse machinery in ibis/client.py
```

Comment:
This docstring is the same as above, yet these classes do different things. Can you fix that?